keyboards/handwired/drevogramr_qmk/readme.md

```diff
@@ -0,0 +1,19 @@
+# Drevo Gramr QMK
+
+![Drevo Gramr 84](https://i.imgur.com/SeMa7B5.jpg)
+
+Drevo Gramr 84 rewired with Arduino (Pro) Micro. Firmware utilizes onboard 74HC595 shift registers for key matrix scanning and onboard LEDs for backlighting. Rewiring schematics can be found [here](https://imgur.com/a/eKutsbr), original keyboard controller should be removed from PCB. Schematics of stock board is [here](https://imgur.com/a/vPOPcIC).
+
+* Keyboard Maintainer: [MAD](https://github.com/wwwMADwww)
+* Hardware Supported: ATMega32U4, Drevo Gramr 84 with PCB labeled as `TB-ZT-S84LED-01-V03 2015-06-12`.
+* Hardware Availability: *AliExpress, Amazon*
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
added